TwilioVerifySDK/TwilioVerify/Sources/Domain/Factor/Models/PushFactor.swift

```diff
@@ -34,4 +34,11 @@ struct PushFactor: Factor, Codable {
 
 struct Config: Codable {
   let credentialSid: String
+  var notificationPlatform: NotificationPlatform = .apn
 }
+
+public enum NotificationPlatform: String, Codable {
+  case apn
+  case none
+}
+
```

Comment:
<table data-meta="generated_by_UnitTests">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/twilio/twilio-verify-ios/blob/2fdfb7d085dc022bdb92dd7135b2ee5defe47bd1/TwilioVerifySDK/TwilioVerify/Sources/Domain/Factor/Models/PushFactor.swift#L44"></span>Files should have a single trailing newline.<br />
<code>trailing_newline</code> <code>PushFactor.swift:44</code></td>
    </tr>
  </tbody>
</table>